### ISSUE TITLE
implement "hossted set auth [appName] false|true" (was: "hossted httpopen appName")

### ISSUE THREAD

**Issue by liorkesos** [hossted/cli#15]
We need to implement this as a command in the cli ( 
```
CMD VERB OBJECT NAME VALUE
hossted set auth appNAme false
hossted set url appName linnovate
hossted set SSL appName sign
```

**Comment by liorkesos** [comment]
```#!/bin/bash

sed -i '/tauth.basicauth.usersfile/d' "/opt/gitbucket/docker-compose.yml"
sed -i -e 's/tauth,//g' "/opt/gitbucket/docker-compose.yml"
sed -i '/.middlewares=tauth/d' "/opt/gitbucket/docker-compose.yml"
sed -i '/installation you may remove/d' "/etc/motd"

cd "/opt/gitbucket/"

docker-compose stop traefik
docker-compose up -d

rm "/opt/gitbucket/letsencrypt/.htpass"
rm $0
```

**Comment by billylkc** [comment]
Set URL

setting := YamlSetting{
        Pattern:  `(PROJECT_BASE_URL=).*$`,
        NewValue: fmt.Sprintf("$1 %s", url), 
}

or **sed** comment or some **shell script**

**Comment by billylkc** [comment]
Available in 0.1.5

**Comment by billylkc** [comment]
I am not sure how exactly the process could be reversed (hossted set auth [appName] true). Can you provide the exact sed commands for the reverse process?

Or is it reversible at all (hossted set auth [appName] true)?

**Comment by niryariv** [comment]
> I am not sure how exactly the process could be reversed (hossted set auth [appName] true). Can you provide the exact sed commands for the reverse process?
> 
> Or is it reversible at all (hossted set auth [appName] true)?

@liorkesos please advise here

**Comment by niryariv** [comment]
@liorkesos need to research ability to handle "set auth true"

**Comment by niryariv** [comment]
handled by commenting/uncommenting a line and restarting @liorkesos add details here

**Comment by billylkc** [comment]
@billylkc I will do the hostted httpopen <AppName> -> **hossted set auth false** first
the hossted set auth true will be fixed later after Lior's research

**Comment by billylkc** [comment]
Created a separate ticket (#32) to handle the set auth true logics (e.g. sed -> comment/uncomments line, etc..)
This ticket will just focus on change httpopen -> set auth false. 

Currently only gitbucket is supported for the command, waiting for @niryariv updates on the General Available commands (#31)

**Comment by billylkc** [comment]
Please test on gitbucket machine (only gitbucket is supported now)
should be able to run **hossted set auth false** in v0.1.8

**Comment by niryariv** [comment]
confirmed to work on GitBucket

**Comment by niryariv** [comment]
Set auth false implemented, set true will be handled in #32
